core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/RepairConfiguration.java

```diff
@@ -207,6 +218,22 @@ public Builder withRepairUnwindRatio(double repairUnwindRatio)
             return this;
         }
 
+        /**
+         * Set the target repair size in bytes.
+         *
+         * This is used to perform sub range repairs within virtual nodes.
+         * The sub ranges will be calculated based on how much data is in the table versus how large the target size
+         * per repair session is.
+         *
+         * @param targetRepairSizeInBytes The target data per repair session
+         * @return The builder
```

Comment:
Nit: Is the @return necessary on all of these builder functions?

Reply:
When you run the javadoc target you get the following if you remove it:

```
[WARNING] ~/workspaces/ecchronos/core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/RepairConfiguration.java:229: warning: no @return
[WARNING] public Builder withTargetRepairSizeInBytes(long targetRepairSizeInBytes)
[WARNING] ^
```